Aptoide/aptoide-client-v8
# [ASV-1455] Update gradle properties ads variables

**What does this PR do?**
This PR aims at putting the name of all ad network variables from gradle properties with the MoPub prefix.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] gradle.properties

**How should this be manually tested?**

After building the app, check if there is any ads related key with 0 as value on the BuildConfig class. This would mean that the names are different in the local gradle properties file and the project gradle properties file.
Also, run the app and check if there are ads to confirm that the ads are working correctly.
 
**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1455](https://aptoide.atlassian.net/browse/ASV-1455)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
